lib/Event/ConfigureOptionsEvent.php

```diff
@@ -12,8 +12,10 @@
 namespace Sulu\Component\DocumentManager\Event;
 
 use Symfony\Component\OptionsResolver\OptionsResolver;
+use Sulu\Component\DocumentManager\DocumentManagerContext;
+use Symfony\Component\EventDispatcher\Event;
 
-class ConfigureOptionsEvent extends AbstractEvent
+class ConfigureOptionsEvent extends Event
```

Comment:
This will break the Debug event dispatcher. Either all events need to extend AbstractEvent (or DebugEvent) or the DebugEventDispatcher needs to handle non-AbstractEvent classes.